docs/aws/Index.rst

```diff
@@ -6,7 +6,7 @@ AWS cloud components
 The nRF Asset Tracker provides a reference implementation of a serverless backend for an IoT product using AWS.
 It is developed using `AWS CDK <https://aws.amazon.com/cdk/>`_ in `TypeScript <https://www.typescriptlang.org/>`_.
 
-The project also provides an easily understandable description of the available (and implemented) features, in a single folder called `features <https://github.com/NordicSemiconductor/asset-tracker-cloud-aws-js/tree/saga/features>`_.
+The project also provides a description of the available (and implemented) features in a folder called `features <https://github.com/NordicSemiconductor/asset-tracker-cloud-aws-js/tree/saga/features>`_ which are used to continuosly verify the implementation using :ref:`Continuous Integration <aws-continuous-integration>`.
```

Comment:
The added subordinate clause is very unclear. Where does it refer to (which are used...)?